Sharprompt/Sharprompt.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+     <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Please fix indent

Reply:
Thank you for your feedback. This should be fixed now.